peripherals/README.md

```diff
@@ -0,0 +1,77 @@
+# Quick start using feagi_agent
+1) `git clone https://github.com/feagi/feagi.git`
+2) `cd feagi/docker`
+3) `docker compose -f feagi.yml build`
+4) Wait until #3 step is complete.
+5) `docker compose -f feagi.yml up`
+6) load it in your preferred browser: `http://localhost:3000/genome/mode`
+7) Click `Sample Genomes`
+8) See Gazebo and brain activity loaded. Feel free to play with your robot!
+
+Any issue? Needs detailed documentation about docker? [Deployment documentation](https://github.com/feagi/feagi/wiki/Deployment)
+
+
+# Where is feagi_agent? 
+`feagi_agent` allows you to integrate with our FEAGI with your preferred robots. 
+`feagi_agent` has been tested on <i>freenove_smart_car, Gazebo, Godot, Tello, and Psychopy.</i> 
+You can use it on your computer or docker.
+
+# What is inside feagi_agent?
+There are two large library: feagi_interface and retina. `feagi_interface.py` handles the bridge between your project and FEAGI. It allows FEAGI to communicate/controls your robot.
+`retina.py` is the vision where it can see things through any type of camera. 
+
+# configuration.py is REQUIRED
+You will need to have your own configuration where you can adjust setting to see the different results. If you don't have one, use the template below:
+```
+app_name = 'gazebo'
```

Comment:
lets plan on making the needed changes to get away from app_name = 'gazebo' this is used for not only gazebo but also Tello and Freenove which will be confusing.

suggesting to change it to 'embodiment'